### DISCUSSION TITLE
Nitrado server restarts using NitrAPI

### DISCUSSION THREAD

**Discussion by Astaroth368** [gameserverapp/Platform#617] [Suggestions]
Please can you add the ability to schedule server restarts etc. for Nitrado users. There’s a PHP library on GitHub: https://github.com/nitrado/NitrAPI-PHP

I’ve already tested it and it works. Users would just need to create a long life token on their Nitrado accounts and enter that in GSA. 

Thanks

**Comment by mvaessen** [comment]
Hi, thanks for your suggestion! I don't have any plans to add such features to RconConnect at the moment. Rconconnect doesn't offer any functionality to start/stop/restart game servers.
We do offer such functionality for dediconnect, which connects with self-hosted / rented machines.

People using nitrado for hosting game servers can use nitrado's web interface to manage their game server.

**Reply by Astaroth368** [reply]
Ok, thanks for looking. One thing I want to add just in case you’re not aware of it, is that the Nitrado web interface doesn’t allow us to send a warning to players. They literally get a 5 second countdown in chat before the server restarts which is obviously no good to anyone. Plus it rolls back to the last save and you can’t schedule a dino wipe with the restart either. Those are the main reasons I made the suggestion. Not sure if that changes anything, but thought it’s worth explaining. 

Thanks

**Reply by mvaessen** [reply]
Thanks for the explanation. I recommend addressing Nitrado's web interface issues with Nitrado, they should be able to fix that. 

For people that want to use GSA's game server manager features I recommend using DediConnect. I have no plans to integrate nitrado's api atm.
